blocks/table/package.json

```diff
@@ -1,93 +1,63 @@
 {
   "name": "@blocks/table",
-  "version": "0.2.1",
-  "private": true,
+  "version": "0.0.0",
```

Comment:
suggest making this `0.0.1`

Reply:
Since previous version of the table block was `0.2.0`, could making it `0.2.1` makes more sense?

Reply:
Eh, I lean towards no, because this is really just a new block that happens to be taking over the name of the old one. It has no relation to it other than the name. I don't feel strongly about it though – maybe @nonparibus has a preference

Reply:
I thought there might be a technical reason to always increase the version while using the same `name` for the package (everything is the same, directory, package name etc.)

Reply:
If anyone was using / relying on the old one I think there would be, but they aren't at the moment. I think restarting the version gives a more accurate description of the development history of this specific block – i.e. it's just started

Reply:
I would personally increment the version number still. I think it is extremely confusing not to do so. A whole rewrite might be a v1 -> v2 jump in ordinary product-land, but with this being pre-v1 I would just go 0.2 -> 0.3

I absolutely don't think the version number should _decrease_ while using the same reserved block name. Recipe for confusion.

Reply:
Please make this `0.3`